subprojects/testing-native/src/main/java/org/gradle/nativeplatform/test/xctest/internal/execution/XCTestExecuter.java

```diff
@@ -55,6 +55,9 @@
  * - Test probing (so we know which tests exist without executing them)
  */
 public class XCTestExecuter implements TestExecuter<XCTestTestExecutionSpec> {
+
+    private TestClassProcessor processor;
```

Comment:
⭕️ Let's revert these changes for now.

Reply:
done, but the `stopNow` methods still need to exist for the interface, so they now throw `UnsupportedOperationException`s.